### ISSUE TITLE
Code health

### ISSUE THREAD

**Issue by 71** [71/dance#93]
A lot of things have moved recently, and the project has had several major refactors since the beginning. As such, the code could be much cleaner. Once more tests are added and we can ensure large changes don't lead to regressions, I'd like to do the following things:
- [x] Find a general convention for defining variables (making wrappers around `registerCommand` or utility functions called from several `registerCommand`s).
- [x] Better name commands, with more descriptive categories and a more obvious mapping between command name and source file name in `src/commands.ts`.
- [x] Clean up `extension.ts`.
- [ ] Reorder imports in all files.

**Comment by tshort** [comment]
I'm curious on release plans. I've been running master since the unstable branch was merged in. It's fantastic! Kudos to @71 and @yuchenshi!

I'm not a power user, but I don't have any major issues, and it fixes many problems with v0.1.9.

**Comment by 71** [comment]
@tshort We were planning on releasing the update on the marketplace but ended up working on other things for a while. Now we're trying to fix issues that won't be too much work and release that on the marketplace, since there haven't been too many issues with `master`.

**Comment by yuchenshi** [comment]
@71 How do you feel about cutting a release after #101 is merged? I think we've got the most low-hanging fruits and we can get more help to find more edge cases if a new version is released.

Also, FYI I imagine objects in caret-based selections is the largest source of bugs. There are tons of edge cases and we don't have great coverage. I've written extensive tests for character-based and used it manually for a while, but not carets. We can also put that in known issues and call for help.

**Comment by 71** [comment]
SGTM. None of us works with `caret` mode, so it will help find errors. It's a significant step up from the previous release so it will be nice to have regardless.

**Comment by 71** [comment]
This was improved in 0.4, and was a major focus of 0.5. Code style is also more strongly enforced.
